docs/usage/guides/cuda/build.sh

```diff
@@ -1,15 +1,27 @@
 #!/bin/bash
-set -e
-cd $(dirname $0)
 
-K3S_TAG="${1:-v1.18.10+k3s1}"
-IMAGE_TAG="${K3S_TAG/+/-}"
+export CI_REGISTRY_IMAGE="YOUR_REGISTRY_IMAGE_URL"
+export VERSION="1.0"
+export K3S_TAG="v1.21.2+k3s1"
+export DOCKER_VERSION="20.10.7"
+export IMAGE_TAG="v1.21.2-k3s1"
 
-if [ -d k3s ]; then
-    rm -rf k3s
+sudo docker build -f Dockerfile.base --build-arg DOCKER_VERSION=$DOCKER_VERSION -t $CI_REGISTRY_IMAGE/base:$VERSION . && \
```

Comment:
Can we drop the `sudo` usage in this script? At the end of the script, docker is also called without sudo and I feel like if someone needs to run docker as sudo (i.e. current user is not in docker group), the whole script can be run as sudo instead.

Reply:
I've removed the sudo commands from the build.sh but I've initially added them so the user 100% won't forget to run it with sudo. I think that the build actually creates k3s/build/out/data.tar.zst file with root permissions because of downstream use of root so you need to use sudo to be able to use it after that.